tests/test_nhg.py

```diff
@@ -1379,6 +1617,507 @@ def test_nhgorch_label_route(self, dvs, testlog):
         self.nhg_ps._del("group1")
         self.asic_db.wait_for_n_keys(self.ASIC_NHG_STR, self.asic_nhgs_count)
 
+class TestCbfNextHopGroup(TestNextHopGroupBase):
+    MAX_NHG_MAP_COUNT = 512
+
+    def test_cbf_nhgorch(self, dvs, testlog):
```

Comment:
Could you add a test case for label route since this CBF NHGMap is used for MPLS as well.

Reply:
I've modified one of the existing cases to use a label route instead of a prefix route